cherrypy/_cpserver.py

```diff
@@ -113,6 +113,8 @@ def _set_socket_host(self, value):
     ssl_private_key = None
     """The filename of the private key to use with SSL."""
 
+    ssl_ciphers = None
```

Comment:
Since cherrypy and python are alll opensource code, we can't add many private behavior to them.
Sometimes, we need to dynamically controled the cipher suites as customer's request.
Like below:
CIPHERS = (
    'ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+HIGH:'
    'DH+HIGH:RSA+AESGCM:RSA+AES:RSA+HIGH:!aNULL:'
    '!eNULL:!MD5:!DSS:!RC4:!DES:!3DES:!BF:!BF+CFB:!BF+ECB:!BF+OFB'
)
              cherrypy.config.update({'tools.secureheaders.on' : True})
              cherrypy.config.update({'server.socket_port': 443})
              cherrypy.config.update({'server.ssl_module': "builtin"})
              cherrypy.config.update({'server.ssl_certificate':"/cfg/cert/management-server.cert"})
              cherrypy.config.update({'server.ssl_private_key':"/cfg/cert/management-server.key"})
              cherrypy.config.update({'server.ssl_ciphers':CIPHERS})